transports/transport-highway/src/main/java/org/apache/servicecomb/transport/highway/HighwayServerVehicle.java

```diff
@@ -31,8 +31,8 @@
 import io.vertx.core.Promise;
 import io.vertx.core.Vertx;
 
-public class HighwayServerVerticle extends AbstractVerticle {
-  private static final Logger LOGGER = LoggerFactory.getLogger(HighwayServerVerticle.class);
+public class HighwayServerVehicle extends AbstractVerticle {
```

Comment:
Why rename this class?

Reply:
@liubao68 That `Verticle` might be a typo

Reply:
`Verticle` is correct, not `Vehicle`